### ISSUE TITLE
Display message from task underneath the task title

### ISSUE THREAD

**Issue by IgnusG** [cenk1cenk2/listr2#1]
The original listr package displays the message from eg. skip underneath the task name.

Listr2 replaces the title with the message:

![image](https://user-images.githubusercontent.com/6438760/77762694-36525180-703a-11ea-83a2-ec1cf2398a07.png)

This leads to a bit of confusion, since the message is in the middle of task names and feels out of place. I didn't find an option to disable this. Is it possible for the renderer to print the skip messages indented and underneath:

![image](https://user-images.githubusercontent.com/6438760/77763397-4c144680-703b-11ea-84e0-4bc64d8d1fa7.png)

I currently use this snippet to achieve this:
```javascript
manager.add({
  title: 'Bump version',
  task: () => {
    if (shouldSkip === true) {
      return new Listr({
        title: "Doesn't matter",
        task: (_, task) => task.skip('Version has been already changed')
      }, { collapse: false })
    }
    return '...!'
  }
});
```

**Comment by cenk1cenk2** [comment]
Yeah i changed that since i wrote my messages as skip messages so i did not really needed the task title anymore. But i can add a flag to configure that behaviour in evening today or tommorrow.

**Comment by cenk1cenk2** [comment]
![image](https://user-images.githubusercontent.com/26881592/77782680-f0a48180-7057-11ea-94b8-1d03aac2ff02.png)
Is this a viable solution for the render?

The default behavior is still to replace the title since it makes more sense for my application. 

It has to be enabled through the option `{ collapseSkips: false }` where `{ collapseSkips: true }` is the default value, for the reason that I tried to explain.

Passing this via manager is much easier.
```typescript
import { Manager } from 'listr2'

  manager.add([
    manager.indent<ListrCtx>((ctx) => [
      ...tasks
  ], { collapseSkips: false })

// or for all tasks
manager.runAll{ collapseSkips: false })

// the use case i use is initiate the class in to another class like the one below so it will work for all cases
this.tasks = new Manager({ renderer: this.getListrRenderer(), collapseSkips: false })
```
I will merge this after I have a look at the other issue. @ #2

**Comment by cenk1cenk2** [comment]
Fixed with [v1.3.5](https://github.com/cenk1cenk2/listr2/releases/tag/v1.3.5), need more testing I did not have the environment for testing at the moment.

**Comment by IgnusG** [comment]
Thanks a lot for the fix @cenk1cenk2!
